runtime/onert/core/src/exec/train/TrainableExecutor.cc

```diff
@@ -82,18 +82,15 @@ void TrainableExecutor::forward(const IODescription &desc, bool training)
                           desc.inputs[i]->size);
   }
 
-  if (!training)
+  // Set output(s)
+  assert(_output_tensors.size() == desc.outputs.size());
+  for (uint32_t i = 0; i < _output_tensors.size(); ++i)
   {
-    // Set output(s)
-    assert(_output_tensors.size() == desc.outputs.size());
-    for (uint32_t i = 0; i < _output_tensors.size(); ++i)
-    {
-      auto tensor = _output_tensors[i];
+    auto tensor = _output_tensors[i];
 
-      if (desc.outputs[i] == nullptr)
-        throw std::runtime_error{"Output " + std::to_string(i) + "'s buffer is not set."};
-      tensor->setUserTensor(static_cast<uint8_t *>(desc.outputs[i]->buffer), desc.outputs[i]->size);
-    }
+    if (desc.outputs[i] == nullptr)
+      throw std::runtime_error{"Output " + std::to_string(i) + "'s buffer is not set."};
```

Comment:
This means does not allow cases where users do not set output buffers. We need to consider when `desc.outputs[i] == nullptr`.

Reply:
Although the user does not set the output buffer, the output description has already been set in the loading model. This code checks whether the output description is null. In any case, the error log is invalid. I will update it.